### PR TITLE
Creating to do list

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
- Installed linters and tailwind css
- added css stylings 
- added dom router
- created the router paths
- created the navbar
- created the TaskForm.jsx file
- created the TaskCard.jsx file
- created the TaskList.jsx file
-  created  the Copyright.jsx file
- created the Home.jsx file
- created the NotFound.jsx file
- added gh-pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,170 @@
...
+<!-- Features -->
 
-### `npm run eject`
+### Key Features <a name="key-features"></a>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- **Informative**
+- **Beautyfull**
```
Comment (IsmaelMastronardi) [inline README.md]: This is just a simple grammar mistake, the word is written as Beautiful.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,170 @@
...
+If you like this project please share with your friends and family!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for teach me and support me to do this
```
Comment (IsmaelMastronardi) [inline README.md]: This is also just a grammar mistake, It should be "thank Microverse for teaching me and supporting me".

**Review by IsmaelMastronardi — COMMENTED** [review]
Hi @IsmaelMastronardi, 

Good job so far👏

# STATUS: APPROVED 💪👏

Congratulations! 🎉

Your project is complete! There is nothing else to say other than... it's time to merge it :ship:



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## suggestions
- Fix README.md grammtical mistakes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

**Comment by Felipe-Perez-Ferraro** [comment]
Thank you for the feedback! 👌

**Pull request merged**